### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by johnzheng1975**
Please provide a description for what this PR is for.
Fixes https://github.com/istio/istio.io/issues/13032

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by johnzheng1975** [comment]
This INGRESS_HOST cannot work until you get real IP, Otherwise, this task cannot complete successfully.  FYI

File: content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 1]
```diff
@@ -171,6 +171,24 @@ EOF
 Finally, follow [these instructions](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)
 to set the `INGRESS_HOST` and `SECURE_INGRESS_PORT` variables for accessing the gateway.
 
+{{< tip >}}
+Note that for some cloud like AWS, you may get INGRESS_HOST like: `ab52747ba608744d8afd530ffd975cbf-330887905.us-east-1.elb.amazonaws.com`, you have to convert it to IP in this task:
```
Comment (johnzheng1975) [inline content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md]:
```suggestion
For some cloud like AWS, you may get INGRESS_HOST with domain: `ab52747ba608744d8afd530ffd975cbf-330887905.us-east-1.elb.amazonaws.com`, you have to convert it to IP in this task:
```

**Comment by ericvn** [comment]
Do the instructions at https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports work with AWS?

```
In certain environments, the load balancer may be exposed using a host name, instead of an IP address. In this case, the ingress gateway’s EXTERNAL-IP value will not be an IP address, but rather a host name, and the above command will have failed to set the INGRESS_HOST environment variable. Use the following command to correct the INGRESS_HOST value:
$ export INGRESS_HOST=$(kubectl -n "$INGRESS_NS" get service "$INGRESS_NAME" -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

**Comment by johnzheng1975** [comment]
> Do the instructions at https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports work with AWS?
> 
> ```
> In certain environments, the load balancer may be exposed using a host name, instead of an IP address. In this case, the ingress gateway’s EXTERNAL-IP value will not be an IP address, but rather a host name, and the above command will have failed to set the INGRESS_HOST environment variable. Use the following command to correct the INGRESS_HOST value:
> $ export INGRESS_HOST=$(kubectl -n "$INGRESS_NS" get service "$INGRESS_NAME" -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
> ```

@ericvn , I test with this docs, it return domain instead of IP.
```
$ INGRESS_NS=istio-system
(eksctltry):  $ INGRESS_NAME=istio-ingressgateway
(eksctltry):  $ export INGRESS_HOST=$(kubectl -n "$INGRESS_NS" get service "$INGRESS_NAME" -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
(eksctltry):  $ echo $INGRESS_HOST
ab52747ba608744d8afd530ffd975cbf-330887905.us-east-1.elb.amazonaws.com
```
Note that with domain, this task will fail. You have to convert domain to IP, to make task successful.

File: content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 2]
```diff
@@ -661,6 +661,14 @@ See [configuring SNI routing](/docs/ops/common-problems/network-issues/#configur
     $ echo "INGRESS_HOST=$INGRESS_HOST, SECURE_INGRESS_PORT=$SECURE_INGRESS_PORT"
     {{< /text >}}
 
+*   For some cloud like AWS, you may get `INGRESS_HOST` with domain.
+    You need convert it to IP:
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md:665]:
```suggestion
*   Make sure the value of `INGRESS_HOST` is an IP address. In some cloud platforms, e.g., AWS, you may
     get a domain name, instead. This task expects an IP address, so you will need to convert it with commands
     similar to the following:
```

**Review by frankbu — APPROVED** [review]

**Comment by frankbu** [comment]
Fixes #13032

**Pull request merged**